TurkishFilter/sections/specific.txt

```diff
@@ -90,6 +90,7 @@ hdfilmcehennemi1.com##a[href*="/click.php"][target="_blank"] > img
 ||mp3indirelim.com/uploads/tvbet_jackpot_300h250_tr.gif
 oyunceviri.net##.jeg_ad
 oyunceviri.net##.adsbygoogle
+nefesgazetesi.com##.logo_area > .banner
```

Comment:
There is also one more ad which is not blocked by this rule (and social leftover).
<details><summary>Screenshot</summary>

![image](https://user-images.githubusercontent.com/29142494/62465067-5624c480-b78e-11e9-9a0b-c565a9ede43f.png)

</details>

Maybe:
`nefesgazetesi.com##a[href^="http://www.nefesgazetesi.com/banner.php"]`
and for social:
`nefesgazetesi.com##.share`